css/css-overflow/WEB_FEATURES.yml

```
@@ -4,29 +4,7 @@ features:
   - scrollbar-gutter-*
 - name: overflow-shorthand
   files:
-  # This is all overflow-* except overflow-auto-scrollbar-gutter-intrinsic-*,
-  # overflow-scroll-*, and overflow-no-interpolation.html which depends on
-  # transition-behavior. TODO: convert to exclusion patterns when possible.
-  - overflow-abpos-transform.html
-  - overflow-body-propagation-*
-  - overflow-canvas.html
-  - overflow-clip-*
-  - overflow-codependent-scrollbars.html
-  - overflow-ellipsis-dynamic-001.html
-  - overflow-empty-child-box.html
-  - overflow-hidden-resize-with-stacking-context-child.html
-  - overflow-img-*
-  - overflow-img.html
-  - overflow-inline-block-with-opacity.html
-  - overflow-inline-transform-relative.html
-  - overflow-negative-margin-dynamic.html
-  - overflow-negative-margin.html
-  - overflow-no-frameset-propagation.html
-  - overflow-overlay.html
-  - overflow-padding.html
-  - overflow-recalc-001.html
-  - overflow-replaced-element-001.html
-  - overflow-shorthand-001.html
-  - overflow-video.html
-  - rounded-overflow-*
-  - select-size-overflow-001.html
+  - overflow-*
+  - !overflow-auto-scrollbar-gutter-intrinsic-*
+  - !overflow-scroll-*
+  - !overflow-no-interpolation.html # depends on transition-behavior
```

Check this comment for more context: https://github.com/web-platform-tests/wpt/pull/47110#discussion_r1693400037

```suggestion
  - "!overflow-auto-scrollbar-gutter-intrinsic-*"
  - "!overflow-scroll-*"
  - "!overflow-no-interpolation.html # depends on transition-behavior"
```